### DISCUSSION TITLE
how do i use this?

### DISCUSSION THREAD

**Discussion by loadingks** [janbaykara/ebook-scraper#49] [Q&A] [answered]
how do i use this?

**Comment by roshho** [comment]
Here's what I found from [u/quitreading ](https://www.reddit.com/r/libgen/comments/szc6s6/getting_a_book_off_of_proquest/hy3ovf8/?context=3) this from [reddit post](https://www.reddit.com/r/libgen/comments/szc6s6/comment/kg1hb4n/?utm_source=share&utm_medium=web2x&context=3). I haven't found success with this so far, but if the creator could give some prelim instructors, that'd be super!

> Here is how to use this ebook scraper (educational purposes only):
> (0. Download node.js onto your computer https://nodejs.org/en to use the "npm" feature mentioned later)
> 1. Login to GitHub
> 2. Download the code by clicking the green "Code" button and download ZIP
> 3. Open the ZIP so that it's uncompressed
> 4. Download VSCode
> 5. In the ZIP you opened, there's a file that says "proquest-ebook-scraper.code-workspace. Click this and it will open in VSCode
> 6. Once in VSCode all the code you downloaded in the ZIP will be open in VSCode
> 7. Open the terminal on the bottom of VSCode. (Google this if you don't know how).
> 8. In the terminal type "npm install webpack". This will download Webpack. You'll see why in the next step.
> 9. In the terminal (after Webpack downloads), type "npm run build". This will use Webpack to "build" the app in a "minified" version
> 10. Once successful, go back to the original folder you opened from the ZIP. You'll find a folder called "dist" with a bunch of code that you generated from the "npm run build".
> 11. Go to Google Chrome > Manage Extensions (or type this in the URL chrome://extensions/)
> 12. Click "load unpacked" and find the "dist" folder. Enter the dist folder and click "open".
> 13. This will create the Chrome Extension.
> 14. Go to proquest or whatever.
> 15. Make sure you're zoomed fully on the textbook so that only one page is loaded on your screen at a time, and so that the textbook is high quality/zoomed in. If it's zoomed out the images it generates will be low quality.
> 16. You can scroll one by one and the chrome extension will gather every page. But I recommend you click the right arrow so it goes page by page. Do it one by one so that it puts the images together.
> 17. Once done, click the Chrome Extension and download it.
> 18. If you loaded the pages weirdly you can go to the Chrome Extension, "show images" and reorder them with the arrows. If you mess up badly, the "reset" button doesn't seem to work. Simply go to the Manage Extensions and delete the Extension. Upload it again to start over.

**Reply by roshho** [reply]
I found this [comment ](https://www.reddit.com/r/libgen/comments/szc6s6/comment/j8lbij7/?utm_source=share&utm_medium=web2x&context=3 )on the [same reddit post](https://www.reddit.com/r/libgen/comments/szc6s6/comment/kg1hb4n/?utm_source=share&utm_medium=web2x&context=3) that archived crx file which you can just drag into chome://extensions

**Comment by janbaykara** [comment, marked as answer]
Just a heads up, you can now install this browser extension directly from the Chrome Web Store: https://chromewebstore.google.com/detail/ebook-scraper/bhoifjhgahfmjkonopmmfifdfjacjeak?authuser=1&hl=en-GB

**Reply by janbaykara** [reply]
Just for some historical interest, usage stats are here: https://chrome-stats.com/d/bhoifjhgahfmjkonopmmfifdfjacjeak/trends

We're currently at 35 users.
